src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,7 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
You should handle the case where user or user.email is undefined to prevent potential errors.